src/Block/Service/BlockServiceInterface.php

```diff
@@ -13,9 +13,45 @@
 
 namespace Sonata\BlockBundle\Block\Service;
 
+use Sonata\BlockBundle\Block\BlockContextInterface;
+use Sonata\BlockBundle\Model\BlockInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
 /**
  * @author Christian Gripp <mail@core23.de>
  */
-interface BlockServiceInterface extends \Sonata\BlockBundle\Block\BlockServiceInterface
+interface BlockServiceInterface
 {
+    /**
+     * @param BlockContextInterface $blockContext
+     * @param Response              $response
```

Comment:
No need for this part

Reply:
Not part of this PR. I want to create atomic PRs with different topic: Remove stuff, Finalize classes/methods, cleanup, ...